src/OrleansAzureUtils/Storage/OrleansSiloInstanceManager.cs

```diff
@@ -241,21 +241,28 @@ private static Uri ToGatewayUri(SiloInstanceTableEntry gateway)
 
         private IEnumerable<SiloInstanceTableEntry> FindAllGatewaySilos()
         {
-            logger.Info(ErrorCode.Runtime_Error_100277, "Searching for active gateway silos for deployment {0} ...", this.DeploymentId);
+            if (logger.IsVerbose) logger.Verbose(ErrorCode.Runtime_Error_100277, "Searching for active gateway silos for deployment {0}.", this.DeploymentId);
```

Comment:
this function is called once every min. It was just creating too much noise in the log.
Instead, it will now print on both success and failure, so if it fails with timeout or otherwise we will know. And if none is printed, it will mean the function was not called at all. So the extra line did not add any new info, but made looking through the log more tedious.